tests/lac/schur_complement_05.with_cxx11=on.with_trilinos=true.with_mpi=true.mpirun=1.output

```diff
@@ -6,10 +6,10 @@ DEAL:ManualSchur:cg::Convergence step 11 value 0
 DEAL:SchurComplementOp:Pre::Starting value 12.7149
 DEAL:SchurComplementOp:Pre::Convergence step 43 value 1.05660e-05
 DEAL:SchurComplementOp:Main::Starting value 7.49608e-06
-DEAL:SchurComplementOp:Main::Convergence step 12 value 4.00858e-10
+DEAL:SchurComplementOp:Main::Convergence step 22 value 2.93110e-10
```

Comment:
Thanks for the tip. I'll look at this a little more closely tomorrow and will report back then. I was trying my hardest to reproduce the results attained with our own solver (the benchmark computation uses `dealii::SolverCG<TrilinosWrappers::MPI::Vector>`), but maybe this is not possible. Trilinos' solvers seem to be a bit more quirky than our own implementations.